rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -373,20 +380,17 @@ void debugger_frame::UpdateUI()
 			if (cpu->state & cpu_flag::dbg_pause)
 			{
 				m_btn_run->setText(RunString);
-				m_btn_step->setEnabled(true);
```

Comment:
Why is this stuff removed all of a sudden

Reply:
Fixed.